cdist/conf/type/__file/gencode-remote

```diff
@@ -91,3 +91,9 @@ case "$state_should" in
         exit 1
     ;;
 esac
+
+if [ -f "$__object/parameter/onchange" ]; then
+   if [ -s "$__messages_out" ]; then
```

Comment:
@4nd3r How do you know that `__file` type wrote to messages_out?
It could do nothing, but types before it wrote to messages_out.
I think you should grep for something or indicate that onchange should happen.

Reply:
right, you are correct. stole idea from #648.